internal/docs/readme.go

```diff
@@ -17,27 +18,62 @@ import (
 	"github.com/elastic/elastic-package/internal/packages"
 )
 
-// ReadmeFile for the Elastic package
-const ReadmeFile = "README.md"
+// ReadmeFile contains file name and status of each readme file.
+type ReadmeFile struct {
+	FileName string
+	UpToDate bool
+	Error    error
+}
+
+// AreReadmesUpToDate function checks if all the .md readme files are up-to-date.
+func AreReadmesUpToDate() ([]ReadmeFile, error) {
+	packageRoot, err := packages.MustFindPackageRoot()
+	if err != nil {
+		return nil, errors.Wrap(err, "package root not found")
+	}
+
+	files, err := ioutil.ReadDir(filepath.Join(packageRoot, "_dev", "build", "docs"))
+	if err != nil && !os.IsNotExist(err) {
+		return nil, errors.Wrap(err, "reading directory entries failed")
+	}
+
+	var readmeFiles []ReadmeFile
+	for _, f := range files {
+		fileName := f.Name()
+		ok, err := isReadmeUpToDate(fileName, packageRoot)
+		if !ok || err != nil {
+			readmeFile := ReadmeFile{
+				FileName: fileName,
+				UpToDate: ok,
+				Error:    err,
+			}
+			readmeFiles = append(readmeFiles, readmeFile)
+		}
+	}
+
+	if readmeFiles != nil {
+		return readmeFiles, fmt.Errorf("check readme files are up-to-date failed")
+	}
+	return nil, nil
```

Comment:
I think it may be beneficial to always return the `readmeFiles` as some kind of a status report (consistency).